content/developers/developer-patterns/navigating-merklelogs/index.md

```diff
@@ -393,42 +320,25 @@ Using our "canonical" mmr log for illustration, we get this diagram
 ```
 
 
-The sibling *proof* path for the leaf with `mmrIndex` 7 would be [8, 12, 6], and
-the "peak bagging" algorithm would then be applied to get the root.
+The sibling *proof* path for the leaf with `mmrIndex` 7 would be [8, 12, 6], and the "peak bagging" algorithm would then be applied to get the root.
 
-A very nice visualisation of how the peaks combine is available in this paper
-on
+A very nice visualization of how the peaks combine is available in this paper on
 [cryptographic, asynchronous, accumulators](https://eprint.iacr.org/2015/718.pdf) (see Fig 4, page 12)
 
-## The "look back" peak stack can be visualised like this
+## The "look back" peak stack can be visualized like this
 
-A specific challenge for log implementations with very large data sets is then
-answering "how far back" or "how far forward" may I need to look ?
+A specific challenge for log implementations with very large data sets is  answering "how far back" or "how far forward" may I need to look?
 
-MMR's differ from classic binary merkle trees in how the incomplete sub trees
-are combined into a common root. For an MMR, the common root is defined by an
-algorithm for combining the, adjacent, incomplete sub trees. Rather than by the
-more traditional, temporary, assignment of un-balanced siblings. Such
-un-balanced siblings would later have to be re-assigned (balanced) when there
+MMRs differ from classic binary Merkle trees in how the incomplete sub trees are combined into a common root. For an MMR, the common root is defined by an algorithm for combining the, adjacent, incomplete sub trees. Rather than by the more traditional, temporary, assignment of un-balanced siblings. Such un-balanced siblings would later have to be re-assigned (balanced) when there
```

Comment:
```suggestion
MMRs differ from classic binary Merkle trees in how the incomplete sub-trees are combined into a common root.
For an MMR, the common root is defined by an algorithm for combining the adjacent, incomplete sub-trees.
Rather than by the more traditional, temporary, assignment of un-balanced siblings.
Such un-balanced siblings would later have to be re-assigned (balanced) when there were sufficient leaves to merge the sub-trees.
```